nodestream_plugin_neptune/neptune_connection.py

```diff
@@ -0,0 +1,115 @@
+import json
+from abc import ABC, abstractmethod
+from logging import getLogger
+
+from aiobotocore.session import get_session
+
+
+class NeptuneConnection(ABC):
+    async def execute(self, query_stmt: str, parameters):
+        response = None
+        async with self._create_boto_client() as client:
+            try:
+                response = await self._execute_query(client,
+                    query_stmt=query_stmt,
```

Comment:
nit: Python formatting best practices:

```
                response = await self._execute_query(client,
                                                     query_stmt=query_stmt,
                                                     parameters=parameters,
                                                     )
```

Reply:
I've rerun `black` and `isort` over the project to ensure consistent formatting.